Makefile

```diff
@@ -55,4 +52,4 @@ testall:
 	@go test ./...
 
 bench:
```

Comment:
Adding here `@echo "Running benchmarks.."`  would make it consistent with other make comments